Exercise_B/index.css

```diff
@@ -2,3 +2,35 @@
 This css sheet is linked to index.html
 You can style tags, classes (.) or ids (#) from this file.
 */
+
+.container{
+    width: 500px;
+    height: 200px;
+}
+
+
+.blue{   
+    background: blue;
+    float: left;
+}
+
+.red{  
+    background: red;  
+    position: relative;
+    left: 200px;
+    top: 100px;
+
+}
```

Comment:
consider using float right for for the green box.
consider using clearfix property for the red box
IF you want to learn more about floats. 

You have replicated the image well so good work!